lib/node_modules/@stdlib/math/base/special/negalucas/src/main.c

```diff
@@ -0,0 +1,123 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/negalucas.h"
+#include "stdlib/math/base/special/abs.h"
+
+static const int32_t STDLIB_CONSTANTS_FLOAT64_MAX_SAFE_NTH_NEGALUCAS = 76;
```

Comment:
Rather than defining it, try using macro at https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/constants/float64/max-safe-nth-lucas.
You may refer https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/fibonacci/src/main.c.

Reply:
@Pranavchiku When i referenced the Max safe nth lucas from the constants, I'm no longer able to get the files to compile.  In the past I've been running the following commands:
make install-node-modules
make examples-c EXAMPLES_FILTER=".*/math/base/special/negalucas/.*"
make benchmark-c BENCHMARKS_FILTER=".*/math/base/special/negalucas/.*"
make install-node-addons NODE_ADDONS_PATTERN=math/base/special/negalucas
make test TESTS_FILTER=".*/math/base/special/negalucas/.*"

is there an intermediate step I need to run after make install-node-modules so the includes can reference constants?